resources/views/components/tools/color.blade.php

```diff
@@ -39,6 +39,20 @@ class="fi-input block w-full border-none py-1.5 text-base text-gray-950 transiti
             <span class="sr-only">{{ trans('filament-tiptap-editor::editor.color.input_label') }}</span>
         </label>
 
+        @if(!empty(config('filament-tiptap-editor.preset_colors')))
+        <div class="mt-2 flex flex-wrap justify-start gap-2">
+            @foreach(config('filament-tiptap-editor.preset_colors') as $name => $value)
+                <span
+                    x-tooltip.raw="{{ $name }}"
+                    class="rounded-full w-5 h-5 cursor-pointer"
+                    style="background-color:{{ $value }};"
+                    x-on:click="setState('{{ $value }}'); editor().chain().focus().setColor(state).run(); $dispatch('close-panel')"
```

Comment:
I don't think clicking the color should close the panel or run the editor updates. It should just update the state then the user still has to click the submit button to actually change the color.